lib/metadata.php

```diff
@@ -45,6 +45,12 @@
 $metadata_object->recordIdField = $record_id_field;
 $metadata_object->dataDictionary = $data_dictionary;
 $metadata_object->events = $event_map;
+if (REDCap::versionCompare ( REDCAP_VERSION , '8.7.1', '<')) {
```

Comment:
This is good to know.

Reply:
Turns out I didn't need to use this, but I thought it was a potentially useful discovery so I left it in. Could be removed though.